schema/package-manifest/schema.json

```diff
@@ -63,7 +63,8 @@
           "type": "string"
         },
         "iconUrl": {
-          "type": "string"
+          "type": "string",
+          "format": "uri"
```

Comment:
The schema file is autogenerated and should not be edited. Unfortunately this isn't well documented yet. 

The source files are in `api/v1alpha1/package_manifest.go` and the schema can be regenerated by calling `make` or `make schema-gen` or, if you don't want to use make, `go run ./internal/cmd/schema-gen -o schema`.

It should be possible to add `jsonschema:"format=uri"` or something like that to the `IconUrl` property of `PackageManifest`. 

This is JSON schema library we use for this: https://github.com/invopop/jsonschema

Reply:
Thanks for the suggestion, will work on it.